core/src/cell.rs

```diff
@@ -31,22 +30,26 @@ impl BlockInfo {
 #[derive(Clone, Eq, PartialEq, Default, Deserialize, Serialize)]
 pub struct CellMeta {
     #[serde(skip)]
-    pub cell_output: Option<CellOutput>,
+    pub cell_output: CellOutput,
     pub out_point: CellOutPoint,
     pub block_info: Option<BlockInfo>,
     pub cellbase: bool,
-    pub capacity: Capacity,
-    pub data_hash: Option<H256>,
+    pub data_bytes: u32,
```

Comment:
I don't understand why not `u64` here, why not keep consistency with `Capacity(u64)`,  and so many convert `u64::from(cell.data_bytes)`